Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
     "mapper/tedge_mapper",
     "mapper/cumulocity/c8y_translator_lib",
     "mapper/thin_edge_json",
+    "hashmap",
```

Comment:
If this is just a prototype then that name is fine. Otherwise not.

Reply:
Yes, its just a PoC